src/sqlfluff/rules/L026.py

```diff
@@ -2,12 +2,18 @@
 
 from sqlfluff.core.rules.analysis.select import get_aliases_from_select
 from sqlfluff.core.rules.base import LintResult
+from sqlfluff.core.rules.doc_decorators import document_configuration
 from sqlfluff.rules.L025 import Rule_L020
 
 
+@document_configuration
 class Rule_L026(Rule_L020):
     """References cannot reference objects not present in FROM clause.
 
+    NB: This rule is disabled by default for BigQuery due to its use of
+    structs which trigger false positives. It can be enabled with the
```

Comment:
Future-looking question (not a "strike" against this PR): Do we believe that someday we could get this rule working better for L026, either working 100% correctly or at least improve it so it can distinguish between:
* Definite issues
* Possible issues

If so, is there (should there be) an issue for that work?

Reply:
No I do not believe this is possible for dialects (like BigQuery) that use structs

It's impossible to distinguish the below as being a non-issue (because `b` is a column reference that is a STRUCT), ot being an issue (because `b` is a missing table alias):

```sql
SELECT 
  a,
  b.c
FROM
  table1
```